js/lib/request.js

```diff
@@ -5,56 +5,34 @@
 'use strict'
 
 const electron = require('electron')
-const BrowserWindow = electron.BrowserWindow
-const ipcMain = electron.ipcMain
-const messages = require('../constants/messages')
-
-const getWebContents = () => {
-  try {
-    return BrowserWindow.getAllWindows()[0].webContents
-  } catch (e) {
-    return null
-  }
-}
-
-var nonce = 0
+const session = electron.session
 
 /**
  * Sends a network request using Chromium's networks stack instead of Node's.
  * Depends on there being a loaded browser window available.
  * @param {string} url - the url to load
- * @param {function} callback - callback to call with the response metadata and
- *   body
  */
 module.exports.request = (url, callback) => {
-  const webContents = getWebContents()
-  if (!webContents) {
-    callback(new Error('Request failed, no webContents available'))
+  let defaultSession = session.defaultSession
```

Comment:
i guess that's fine for now. in the future, if `request` is handling GET requests that contains info about private browsing (ex: for the ledger), it would be better to not write to disk